vehicles/Steering/gimbal.cc

```diff
@@ -164,6 +166,9 @@ static control::Gimbal* gimbal = nullptr;
 static control::gimbal_data_t* gimbal_param = nullptr;
 static bsp::Laser* laser = nullptr;
 
+osEventFlagsId_t gimbal_motor_Event = osEventFlagsNew(nullptr);
```

Comment:
Don't mix CamelCase and linux_case

Also for `gimbal_motor_Event` and `keymap_Error_Event`